nrf52840.resc

```diff
@@ -0,0 +1,18 @@
+:name: NRF52840
+:description: This script runs Zephyr Shell demo on NRF52840.
```

Comment:
`This script runs a sample Zephyr application on nRF52840.`

Reply:
Done.